features/attached_enable.feature

```diff
@@ -127,8 +127,8 @@ Feature: Enable command behaviour when attached to an UA subscription
             ESM Infra is already enabled.
             See: sudo ua status
             """
-        When I run `apt install -y <pkg-version>` with sudo
```

Comment:
When unspecified exit code in our When clause, I think we should actually default to check for non-zero exit and fail otherwise. I don't think we should require any call to append the "exits '0`' suffix.